### ISSUE TITLE
drag event stick when mouseup.

### ISSUE THREAD

**Issue by jechav** [ganlanyuan/tiny-slider#101]
Might not look as a bug but seems rare when click on on slide and the drag event still active.
whe should use a variable to control this. 

```js
dragActive = false;

function onTouchOrMouseStart(e) {
  dragActive = true;
...
}

function onTouchOrMouseMove(e) {
  if (!dragActive) return false;
...
}

function onTouchOrMouseEnd(e) {
    dragActive = false;
...
}
```

**Comment by ganlanyuan** [comment]
Would you let me know in which circumstance this happens?

**Comment by jechav** [comment]
@ganlanyuan Hi, 
look https://youtu.be/s8An96fKKZY i made a little video, is not a big problem but that behavior is not expected i think.
When a i click a slide the drag event is not released immediately and the picture move with the cursor until the slide has changed.

------------------------

i am also working in a wrapper for react of your plugin. https://github.com/jechav/tiny-slider-react
now it's in early stage. let me know what do you think.

thanks so much.

**Comment by ganlanyuan** [comment]
Hey, 
I added a [fix](https://github.com/ganlanyuan/tiny-slider/issues/67#issuecomment-342773555) yesterday. Could you check that?

> i am also working in a wrapper for react of your plugin. https://github.com/jechav/tiny-slider-react
now it's in early stage. let me know what do you think.

Sounds great!

**Comment by ganlanyuan** [comment]
Same with #67

**Comment by ganlanyuan** [comment]
Fix added in [v2.3.3](https://github.com/ganlanyuan/tiny-slider/releases/tag/v2.3.3)

**Comment by jechav** [comment]
yea thanks 👍
